terraspace/app/modules/eks-auth-sync/main.tf

```diff
@@ -0,0 +1,90 @@
+terraform {
+  required_providers {
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "~> 2.7.1"
+    }
+
+    helm = {
+      source  = "hashicorp/helm"
+      version = "~> 2.4.1"
+    }
+  }
+  required_version = ">= 1.0.0"
+}
+
+resource "kubernetes_config_map" "eksauth" {
+  metadata {
+    name      = "eks-auth-sync"
+    namespace = var.namespace
+  }
+  data = {
+    "config.yaml" = "${templatefile("${path.module}/config.yaml", { account_id = local.account_id, cluster_name = var.cluster_name })}"
+  }
+}
+
+
+resource "kubernetes_cron_job" "eksauth" {
+  metadata {
+    name      = "eks-auth-sync"
+    namespace = var.namespace
+  }
+  spec {
+    schedule = var.cronjob_schedule
+    job_template {
+      metadata {}
+      spec {
+        completions                = 1
+        parallelism                = 1
+        backoff_limit              = 3
+        active_deadline_seconds    = 120
+        ttl_seconds_after_finished = 300
+        template {
+          metadata {}
+          spec {
+            container {
+              name = "eks-auth-sync"
+              # image   = "registry.gitlab.com/polarsquad/eks-auth-sync" # TODO: Return when fork PR is accepted ## https://gitlab.com/polarsquad/eks-auth-sync/-/merge_requests/9
+              image             = "ghcr.io/francardoso93/eks-auth-sync"
```

Comment:
looks like that MR has been merged.  Might want to include that change here, or spin up another task to do it after we get this PR merged.